v2/emailpassword/pre-built-ui/setup/frontend.mdx

```diff
@@ -586,14 +590,15 @@ Add the highlighted code snippet to your root level `render` function.
 
 ```tsx
 import React from 'react';
+import { ThirdPartyEmailPasswordPreBuiltUI } from "supertokens-auth-react/recipe/thirdpartyemailpassword/preBuiltUI";
 import SuperTokens, { SuperTokensWrapper } from "supertokens-auth-react";
 
 class App extends React.Component {
     render() {
         // highlight-start
-        if (SuperTokens.canHandleRoute()) {
+        if (ThirdPartyEmailPasswordPreBuiltUI.canHandleRoute()) {
```

Comment:
why is thirdpartyemailpassword being used in emailpassword recipe?

Reply:
Please try and change these to use variables wherever possible